cli/cli.py

```diff
@@ -585,11 +694,13 @@ def desbordante_cli(**kwargs: Any) -> None:
     check_error_option_presence(curr_task, error_opt)
     check_error_measure_option_presence(curr_task, error_measure_opt)
 
+    opts = process_tables_options(kwargs)
+
     start_point = process_time()
-    used_opts = set_algo_options(curr_algo, kwargs)
+    used_opts = set_algo_options(curr_algo, opts)
     curr_algo.load_data()
-    used_opts |= set_algo_options(curr_algo, kwargs)
-    provided_options = get_provided_options(kwargs)
+    used_opts |= set_algo_options(curr_algo, opts)
+    provided_options = get_provided_options(opts)
```

Comment:
If a user executes `./cli.py --tables-list ... --tables-directory ...`, but the algorithm doesn't use the "tables" option, it looks like this will output `Unused options: {"tables"}`, which is confusing.